doc/classes/Object.xml

```diff
@@ -59,6 +59,21 @@
 				Sets a property. Returns [code]true[/code] if the [code]property[/code] exists.
 			</description>
 		</method>
+		<method name="_to_string">
```

Comment:
This should have `qualifiers="virtual"` if you use `godot --doctool .` to generate it.